protocol/contracts/v2/TreasuryReservesVault.sol

```diff
@@ -174,11 +181,20 @@ contract TreasuryReservesVault is ITreasuryReservesVault, TempleElevatedAccess {
         StrategyConfig storage strategyConfig = strategies[strategy];
         strategyConfig.underperformingEquityThreshold = underperformingEquityThreshold;
 
-        address _token;
+        mapping(IERC20 => uint256) storage _tokenCredits = strategyTokenCredits[strategy];
+
+        uint256 _ceiling;
+        IERC20 _token;
         uint256 _length = debtCeiling.length;
         for (uint256 i; i < _length; ++i) {
-            _token = debtCeiling[i].asset;
-            strategyConfig.debtCeiling[IERC20(_token)] = debtCeiling[i].balance;
+            _token = IERC20(debtCeiling[i].asset);
+            _ceiling = debtCeiling[i].balance;
+
+            // Note: This will be redundant when https://github.com/yAudit/temple-lending-report/issues/11 is addressed.
```

Comment:
Cleaned up in https://github.com/TempleDAO/temple/pull/842/commits/0fbaab5432cadf348ecc9fc70c6a6fe03f3b1bfb